apps/server/src/shared/repo/base-domain-object.repo.ts

```diff
@@ -2,7 +2,7 @@ import { EntityData, EntityName, FilterQuery, Primary, RequiredEntityData, Utils
 import { EntityManager } from '@mikro-orm/mongodb';
 import { Injectable, InternalServerErrorException } from '@nestjs/common';
 import { AuthorizableObject, DomainObject } from '@shared/domain/domain-object';
-import { BaseEntity, baseEntityProperties } from '@shared/domain/entity';
+import { BaseEntity, baseEntityProperties } from '@shared/domain/entity/base.entity';
```

Comment:
```suggestion
import { BaseEntity, baseEntityProperties } from '@shared/domain/entity';
```

???

Reply:
See the other comment. Because of ALL_ENTITES it leads to dependency cycles